content/notes/create-wp-redis-instance.md

```diff
@@ -23,9 +27,10 @@ cd /var/www/html
 wp-cli plugin install --activate redis-cache
 
 # Define Redis path
-wp-cli config set WP_REDIS_PATH /var/tmp/redis.sock
-wp-cli config set WP_REDIS_HOST /var/tmp/redis.sock
 wp-cli config set WP_REDIS_SCHEME unix
+wp-cli config set WP_REDIS_PATH /var/www/redis.sock
+wp-cli config set WP_REDIS_HOST /var/www/redis.sock
```

Comment:
This is bogus configuration, likely to result in inconsistent or undefined behavior in the future. _PATH supersedes _HOST which assumes the type is a TCP socket, not UDS. See [Redis-Object-Cache/README.md](https://github.com/Sneezry/Redis-Object-Cache/blob/master/README.md#connection-parameters).